packaging/os/yum.py

```diff
@@ -677,6 +683,13 @@ def install(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos):
         pkgs.append(pkg)
 
     if pkgs:
+        
+        if(module.params['excludedocs']):
+            # Needed for the --tsflags nodocs option.
+            module.run_command(['yum', 'install', "yum-tsflags", '-y'], check_rc=True) 
```

Comment:
Don't install the yum-tsflags package here. Instead put it in `requirements` and mention it in the documentation.
Then people can choose whether they want to use this feature and flag or not.